core/unit_test/TestMathematicalFunctions.hpp

```diff
@@ -424,7 +424,7 @@ DEFINE_BINARY_FUNCTION_EVAL(copysign, 1);
     template <typename T, typename U, typename V>                             \
     static KOKKOS_FUNCTION auto eval(T x, U y, V z) {                         \
       static_assert(                                                          \
-          std::is_same<decltype(Kokkos::FUNC((T)0, (U)0), (V)0),              \
+          std::is_same<decltype(Kokkos::FUNC((T)0, (U)0, (V)0)),              \
```

Comment:
Uhoh ...

Reply:
The good old comma operator...

Reply:
Was going through the 2-arg version...  Thanks for fixing